packages/govuk-frontend/src/govuk/template.jsdom.test.js

```diff
@@ -94,6 +94,18 @@ describe('Template', () => {
       expect(document.documentElement).toHaveClass('govuk-template--rebranded')
     })
 
+    it('adds the rebrand class if govukRebrand is set to true as a nunjucks global', () => {
```

Comment:
Reckon it's worth adding a test that covers `govukRebrand` being a function?

I think just adding one single test for that should be fine, as we're using the same approach everywhere. This seems like the logical place for it.